docs/platforms/react-native/integrations/default.mdx

```diff
@@ -109,75 +78,42 @@ Available options:
 
 Here's a code example of how this could be implemented:
 
-<PlatformContent includePath="configuration/linked-errors" />
+```javascript
+<Button
+  title="Get Reviews"
+  onPress={async () => {
+    const movie = event.target.dataset.title;
+    try {
+      await fetchMovieReviews(movie);
+    } catch (e) {
+      const fetchError = new Error(`Failed to fetch reviews for: ${movie}`);
+      fetchError.cause = e;
```

Comment:
NIT if users tries this snippet, cause is not exposed by default so it could lead to a build error.
```suggestion
      // @ts-expect-error cause is not exposed on default error class.
      fetchError.cause = e;
```